### ISSUE TITLE
Suggestion: Make slots in the breeders validate inputs before adding items to the inventory

### ISSUE THREAD

**Issue by Raidobw2** [CreeperHost/Chickens#20]
Hi! I'm not sure if this might already have been taken care of since I didn't see issues about it. I also did not test with the latest 1.19.2.

### Summary

Automation pipes putting random items into the breeders can make a server crash

### Links

The issue at https://github.com/FTBTeam/FTB-Modpack-Issues/issues/1347 states:
> Seed slot should only accept seeds. Input Slots should only accept chickens. Product slots should not be available to insert into. Illegal items in slots should be wiped on startup, should not cause crash.

With attached crash text:
```
java.lang.NullPointerException: Cannot invoke "net.creeperhost.chickens.registry.ChickensRegistryItem.isEnabled()" because "parent1" is null
	at net.creeperhost.chickens.registry.ChickensRegistry.getChildren(ChickensRegistry.java:69) ~[chickensmod-1.19-1.0.7.jar%23167!/:1.0.7] {re:classloading,pl:epic_rei_lmao:B}
	at net.creeperhost.chickens.registry.ChickensRegistry.getRandomChild(ChickensRegistry.java:134) ~[chickensmod-1.19-1.0.7.jar%23167!/:1.0.7] {re:classloading,pl:epic_rei_lmao:B}
	at net.creeperhost.chickens.blockentities.BlockEntityBreeder.tick(BlockEntityBreeder.java:128) ~[chickensmod-1.19-1.0.7.jar%23167!/:1.0.7] {re:classloading,pl:epic_rei_lmao:B}
	at net.creeperhost.chickens.block.BlockBreeder.lambda$getTicker$0(BlockBreeder.java:54) ~[chickensmod-1.19-1.0.7.jar%23167!/:1.0.7] {re:classloading,pl:epic_rei_lmao:B}
	at net.minecraft.world.level.chunk.LevelChunk$BoundTickingBlockEntity.m_142224_(LevelChunk.java:687) ~[client-1.19-20220607.102129-srg.jar%23305!/:?] {re:classloading,pl:epic_rei_lmao:B}
	at net.minecraft.world.level.chunk.LevelChunk$RebindableTickingBlockEntityWrapper.m_142224_(LevelChunk.java:780) ~[client-1.19-20220607.102129-srg.jar%23305!/:?] {re:classloading,pl:epic_rei_lmao:B}
	at net.minecraft.world.level.Level.m_46463_(Level.java:446) ~[client-1.19-20220607.102129-srg.jar%23305!/:?] 
```

### Versions
`chickensmod-1.19-1.0.7.jar`
In FTB One ( a MC 1.19 modpack, which will jump to 1.19.2 at some point)
Forge 41.0.63

### Additional info
Thanks in advance 🙂

**Comment by gigabit101** [comment]
This is already fixed in 1.18, will port these fixes over to 1.19 when I'm done

**Comment by gigabit101** [comment]
Fixed in 1.19.2-1.0.30
